examples/operator/basic-watcher/README.md

```diff
@@ -1,6 +1,7 @@
-# Basic Example
+# Basic Watcher Example
 
-This example code is a super-basic one-file example of a very simple kubernetes custom resource operator using the SDK. It doesn't handle synchronization or missed events, and is extremely noisy due to not filtering updates.
+This example code is a super-basic one-file example of a very simple kubernetes custom resource operator with a Watcher, using the SDK. 
+It doesn't handle synchronization or missed events, and is extremely noisy due to not filtering updates.
```

Comment:
This actually isn't true for `simple.Operator.WatchKind`, which wraps the provided `simple.Watcher` in an `operator.OpinionatedWatcher`.

Reply:
Yeah I still have to review the readmes 😬